docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -176,4 +176,4 @@ To learn more about topics mentioned in this tutorial, visit the following resou
 - [Model Builder Scenarios](../automate-training-with-model-builder.md#scenarios)
 - [Regression](../resources/glossary.md#regression)
 - [Regression Model Metrics](../resources/metrics.md#metrics-for-regression)
-- [NYC TLC Taxi Trip data set](http://www.nyc.gov/html/tlc/html/about/trip_record_data.shtml)
+- [NYC TLC Taxi Trip data set](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
```

Comment:
ℹ️ This site has been redesigned; the URL auto-forwards to this updated URL, which is https compliant.

Reply:
I think it's a great improvement since the site auto-redirects to this new URL.